tests/Fixer/Semicolon/SemicolonAfterInstructionFixerTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Semicolon;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ */
+final class SemicolonAfterInstructionFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @dataProvider provideCases
+     * @requires PHP 5.4
```

Comment:
>isn't whole thing more about short_open_tag instead of PHP version ?

it is, I look at;
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.11/Symfony/CS/Tests/Fixer/Contrib/ShortEchoTagFixerTest.php#L23

for a quick fix, but will refactor!